src/Native/System.Native/pal_process.cpp

```diff
@@ -346,3 +356,75 @@ int32_t WTermSig(int32_t status)
 {
     return WTERMSIG(status);
 }
+
+static
+int32_t ConvertPathConfNameToPlatform(PathConfNames& name)
+{
+    switch (name)
```

Comment:
I'd inline this into PathConf as I did with SysConf, we don't need to convert these for any other reason.